cypress.json

```diff
@@ -0,0 +1,3 @@
+{
+  "baseUrl": "https://localhost:8000"
```

Comment:
Is this necessary when we set as `env` value on the e2e job? Also, doesn't this change local/dev behavior?

Reply:
The cypress.json itself is needed cypress will not run if it's missing. I've omitted the baseUrl property, as we use env values instead. See 4a7d28e.